internal/storage/db.go

```diff
@@ -76,3 +82,39 @@ func (s *DBStorage) AssignTokens(ctx context.Context, issuanceWeek, firstAutomat
 	_, err = s.DBS.DBS().Writer.ExecContext(ctx, assignTokensQuery, issuanceWeek, weekLimit.String())
 	return err
 }
+
+var tokensPerWeekQuery = `
+SELECT 
+	SUM(r.tokens)::numeric / NULLIF(SUM(
+        r.streak_points + 
+        r.integration_points + 
+        r.aftermarket_device_points + 
+        r.synthetic_device_points
+    ), 0) * $2 as tokens_for_points
+FROM rewards_api.issuance_weeks w
+JOIN rewards_api.rewards r TABLESAMPLE BERNOULLI(10)
+    ON r.issuance_week_id = w.id
+WHERE w.starts_at <= $1 
+AND w.ends_at > $1
+AND r.tokens IS NOT NULL
+AND r.tokens > 0;
```

Comment:
I honestly don't think we populate `tokens` anymore.

Reply:
Like if you look here it's all itemized now https://github.com/DIMO-Network/rewards-api/blob/19597aeb94b3c42d2a11368211de7b0a35db4dbf/internal/storage/db.go#L37